tests/test_weighted_pandas.py

```diff
@@ -817,6 +817,8 @@ def test_AreaPlot(mcmc_df, mcmc_wdf):
 
     assert_allclose(axes_df.get_xlim(), axes_wdf.get_xlim(), rtol=1e-3)
     assert_allclose(axes_df.get_ylim(), axes_wdf.get_ylim(), rtol=1e-3)
+    assert_allclose(axes_df.get_ylim()[0], 0.0, rtol=1e-3)
+    assert_allclose(axes_wdf.get_ylim()[0], 0.0, rtol=1e-3)
```

Comment:
> @williamjameshandley @lukashergt do you think it's appropriate to explicitly check that the lower y limit is 0?

Hmm, this is testing for pandas default settings and not really for anesthetic, so I don't think this is what we want...

Reply:
removed